examples/using_smirnoff_in_amber_or_gromacs/export_with_interchange.ipynb

```diff
@@ -1,21 +1,12 @@
 {
```

Comment:
<p>(blocking) ditto (see if you can regenerate outputs without this warning)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>